platforms/karaf/features/src/main/resources/features.xml

```diff
@@ -268,6 +268,7 @@
     <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.azure-core/${azure-core-bundle-version}</bundle>
     <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.azure-storage-common/${azure-storage-common-bundle-version}</bundle>
     <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.azure-storage-blob/${azure-storage-blob-bundle-version}</bundle>
+    <bundle dependency='true'>mvn:org.apache.servicemix.bundles/org.apache.servicemix.bundles.azure-storage-blob-changefeed/12.0.0-beta8_1-SNAPSHOT</bundle>
```

Comment:
Should the version moved to the `pom.xml`?

Reply:
Yes, as you can see in the description/comment, I will use a property and correct released version when bundle release is done.